docs/_docs/05_troubleshooting/02-open-telemetry.md

```diff
@@ -22,24 +22,24 @@ Concepts:
 
 Tracing configuration:
 
-- `TERRAGRUNT_TELEMETRY_TRACE_EXPORTER` - traces exporter type to be used. Currently supported values are:
+- `TG_TELEMETRY_TRACE_EXPORTER` - traces exporter type to be used. Currently supported values are:
   - `none` - no trace exporting, default value.
   - `console` - to export traces to console as JSON
   - `otlpHttp` - to export traces to an OpenTelemetry collector over HTTP [otlptracehttp](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp)
   - `otlpGrpc` - to export traces over gRPC [otlptracegrpc](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc)
   - `http` - to export traces to a custom HTTP endpoint using [otlptracehttp](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp)
-- `TERRAGRUNT_TELEMETRY_TRACE_EXPORTER_HTTP_ENDPOINT` - in case of `http` exporter, this is the endpoint to which traces will be sent.
-- `TERRAGRUNT_TELEMETRY_TRACE_EXPORTER_INSECURE_ENDPOINT` - if set to true, the exporter will not validate the server's certificate, helpful for local traces collection.
-- `TRACEPARENT` - if set, the value will be used as a parent trace context, format `TRACEPARENT=00-<hex_trace_id>-<hex_span_id>-<trace_flags>`, example: `TRACEPARENT=00-xxx-yyy-01`
+- `TG_TELEMETRY_TRACE_EXPORTER_HTTP_ENDPOINT` - in case of `http` exporter, this is the endpoint to which traces will be sent.
+- `TG_TELEMETRY_TRACE_EXPORTER_INSECURE_ENDPOINT` - if set to true, the exporter will not validate the server's certificate, helpful for local traces collection.
+- `TG_TELEMETRY_TRACEPARENT` - if set, the value will be used as a parent trace context, format `TG_TELEMETRY_TRACEPARENT=00-<hex_trace_id>-<hex_span_id>-<trace_flags>`, example: `TG_TELEMETRY_TRACEPARENT=00-xxx-yyy-01`
```

Comment:
Make sure you revert this change when you make the update to the name of the env var.

Reply:
Ok.